oas_worker/devset/README.md

```diff
@@ -0,0 +1,32 @@
+# Devset
+
+Devset is our own development dataset to evaluate transcript quality and keyword extraction performance. It actually consists of 20 curated and manually labeled samples from [cba](https://cba.fro.at/) and should grow in size in the near future. You can find the dataset as `Devset.csv` (as well as a RSS feed generated consisting of its samples: `rss.xml`) in `oas_worker/devset/assets` directory.
+
+
+## Evaluation
+
+You can do an evaluation run either on our OAS devset or generate your own devset feed (which is described in the below section). Follow these steps to run evaluation on the OAS devset:  
+1. Serve OAS devset feed: run in `oas_worker/devset/` this command: `sh serve_nlp_devset.sh`.
```

Comment:
TODO but can come later: Let's add a script that does these 4 steps automatically so that we can run it automated or in CI.